source/navitiacommon/navitiacommon/models/__init__.py

```diff
@@ -583,6 +583,8 @@ class Instance(db.Model):  # type: ignore
     external_services = db.relationship(
         "ExternalService", secondary=associate_instance_external_service, backref="instances", lazy='joined'
     )
+    # max_waiting default value 4h: 4*60*60 = 14400 minutes
+    max_waiting = db.Column(db.Integer, nullable=False, server_default='14400')
```

Comment:
can you use navitiacommon/default_values.py/max_waiting_duration instead of hard-coding  the default value at two different place ?

Reply:
:+1: